connection.js

```diff
@@ -169,8 +172,10 @@ function Connection(client, server) {
     this.transaction = null;
     this.tran_count = 0;
     this.capabilities = null;
-    this.early_talker_delay = config.get('early_talker.pause') || config.get('early_talker_delay') || 1000;
-    this.banner_includes_uuid = config.get('banner_includes_uuid') ? true : false;
+    this.early_talker_delay = config.get('early_talker.pause') ||
+                              config.get('early_talker_delay') || 1000;
+    this.banner_includes_uuid =
+        config.get('banner_includes_uuid') ? true : false;
```

Comment:
Sweet. So I guess we need to specify that these things are boolean in the
get() call.

On Sun, Feb 8, 2015 at 11:25 PM, Matt Simerson <notifications@github.com>
wrote:

> In connection.js
> <https://github.com/baudehlo/Haraka/pull/831#discussion_r24308584>:
>
> > @@ -169,8 +172,10 @@ function Connection(client, server) {
> >      this.transaction = null;
> >      this.tran_count = 0;
> >      this.capabilities = null;
> > -    this.early_talker_delay = config.get('early_talker.pause') || config.get('early_talker_delay') || 1000;
> > -    this.banner_includes_uuid = config.get('banner_includes_uuid') ? true : false;
> > +    this.early_talker_delay = config.get('early_talker.pause') ||
> > +                              config.get('early_talker_delay') || 1000;
> > +    this.banner_includes_uuid =
> > +        config.get('banner_includes_uuid') ? true : false;
>
> I believe what you're asking for is implemented right here
> <https://github.com/baudehlo/Haraka/blob/master/configfile.js#L217>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/baudehlo/Haraka/pull/831/files#r24308584>.
>